presto-docs/src/main/sphinx/installation/deployment.rst

```diff
@@ -10,6 +10,30 @@ Linux Operating System
 
 * 64-bit required
 * newer release preferred, especially when running on containers
+* adequate ulimits for the user that runs the Presto process. These limits 
+  may depend on the specific Linux distribution you are using. Currently,
+  Presto requires a minimum of 4096 open file descriptors, and strongly
+  recommends at least 8192 when it runs. However, the number of open file
+  descriptors needed for a particular Presto instance scales with the number
+  of machines in the cluster as roughly ``number of machines^2 * workload``.
```

Comment:
`number of machines^2` is hard to read (normally there would be a space between operators, or it would be a single word). Also, it's not clear what workload means. I think we could simplify this

> as roughly as the square of the machine count, times some factor depending on the workload